src/models.rs

```diff
@@ -17,6 +17,7 @@ use crate::finding::{Route, SymbolicLocation};
 /// providing access to the underlying data model.
 pub(crate) struct Workflow {
     pub(crate) path: String,
+    pub(crate) contents: String,
```

Comment:
I might be missing something, but why do we need another copy of the YAML document here? `workflow.document.source()` should return a borrowed (`&str`) copy that should be reusable here :slightly_smiling_face:

Reply:
You definetely did not miss anything, I actually missed that `document.source()` API! Super handy! I'll change that

Reply:
Fixed [here](https://github.com/woodruffw/zizmor/pull/187/commits/0de3177ed5dd2f3e944b3e2edc527742e4531a5e)

Reply:
Thanks!